.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+## Upgrade Notes
+Does this PR prevent a zero down-time upgrade? (Assume upgrade order: Controller, Broker, Server, Minion)
+* [ ] Yes (Please label as **<code>backward-incompat</code>**)
+
+Does this PR fix a zero-downtime upgrade introduced earlier?
+* [ ] Yes (Please label this as **<code>backward-incompat</code>**)
+
+Does this PR otherwise need attention when creating release notes? Things to consider:
+- New configuration options
+- Deprecation of configurations
+- Signature changes to public methods/interfaces
+- New plugins added or old plugins removed
+* [ ] Yes (Please label this PR as **<code>release-notes</code>**)
```

Comment:
Even with the "Things to consider", the release notes section is still just a Yes/No question. When the answer is Yes, better explicitly ask the author to prepare some release notes, to make the release master's life easier.

e.g. (borrowing the norm from Presto)

```
== RELEASE NOTES ==

General Changes
* ...
* ...
```

If release note is NOT required, use:

```
== NO RELEASE NOTE ==
```

Reply:
@haibow  added, please verify